packages/core/types/@chainlink/index.d.ts

```diff
@@ -93,6 +93,7 @@ declare module '@chainlink/types' {
     returnRejectedPromiseOnError?: boolean
     verbose?: boolean
     api?: RequestConfig
+    DEFAULT_ENDPOINT?: string
```

Comment:
Could we maintain casing convention and re-name this to `defaultEndpoint`?

TypeScript and VSCode is powerful as hell because you can highlight `DEFAULT_ENDPOINT` and hit f2 and rename it everywhere at once.